storage/ctab/src/main/java/org/openscience/cdk/io/CharIter.java

```diff
@@ -0,0 +1,100 @@
+package org.openscience.cdk.io;
+
+/**
+ * Helper class that facilitates the parsing of text data.
+ */
+final class CharIter {
+    private final String string;
+    private int position = 0;
+
+    CharIter(String str) {
+        this.string = str;
+    }
+
+    static boolean isSpace(char c) {
```

Comment:
Helper classes should also have documentation on public methods.

Reply:
I am always happy to write a bit of javadoc, but this method isn't public but has package level access, i.e. if using default settings it wouldn't actually turn up in the generated html.